test/unit/oatbox/log/LoggerServiceTest.php

```diff
@@ -1,166 +1,168 @@
 <?php
-/*  
+/*
  * This program is free software; you can redistribute it and/or
  * modify it under the terms of the GNU General Public License
  * as published by the Free Software Foundation; under version 2
  * of the License (non-upgradable).
- * 
+ *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  * GNU General Public License for more details.
- * 
+ *
  * You should have received a copy of the GNU General Public License
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
- * 
+ *
  * Copyright (c) 2008-2010 (original work) Deutsche Institut für Internationale Pädagogische Forschung (under the project TAO-TRANSFER);
  *               2009-2012 (update and modification) Public Research Centre Henri Tudor (under the project TAO-SUSTAIN & TAO-DEV);
- * 
+ *
  */
 namespace oat\generis\test\unit\oatbox\log;
 
+use Monolog\Handler\StreamHandler;
+use oat\oatbox\log\logger\TaoMonolog;
 use oat\oatbox\log\LoggerService;
 use oat\oatbox\log\logger\TaoLog;
 use oat\generis\test\TestCase;
+use Psr\Log\LogLevel;
 
-class LoggerServiceTest extends TestCase {
-	
-	const RUNS = 1000;
-    
-    protected function setUp()
+class LoggerServiceTest extends TestCase
+{
+    /** @var LoggerService */
+    private $subject;
+
+    /** @var string */
+    private $logFilePath;
+
+    protected function setUp(): void
+    {
+        $this->logFilePath = tempnam(sys_get_temp_dir(), 'logtest');
```

Comment:
This test looks more like an integration test that involves `SingleFileAppender` and writes data to a disk. As I understand, we should mock dependencies in unit tests, it will simplify it. `SingleFileAppender` should have own unit test.

Reply:
Please consider https://github.com/oat-sa/tao-core/blob/master/test/TaoPhpUnitTestRunner.php#L141 as an abstraction for FS related operations for tests, if possible.